string debug
string drone_id
uint8 code 0 # faster way for drone to leader/gs to check/compare status

# timestamp # if useful
